BREEZE.rst

```diff
@@ -788,6 +788,16 @@ For example this will only run provider tests for airbyte and http providers:
 
    breeze testing tests --test-type "Providers[airbyte,http]"
 
+You can also exclude some providers from ``breeze testing tests`` command is that you can easily
+specify to run all Providers except those specified.
```

Comment:
I am not english native speaker so I might be wrong but the sentence looks weird to me.

```suggestion
You can also exclude some providers from ``breeze testing tests`` command  to run all providers except those specified.
```

Reply:
It is indeed :)